src/components/Layout/Footer.js

```diff
@@ -20,11 +29,12 @@ const Footer = ({ t }) => {
             <Grid.Column>
               <Header inverted as="h5" floated="left">
                 {t('nav.top.header')}
-                <br />
+                <br/>
                 <small className="text grey">
-                  {t('nav.footer.copyright', { year })}
+                  {t('nav.footer.copyright', {year})}
                   {' '}
                   {t('nav.footer.rights')}
+                  {t('nav.footer.bottomTextStart') && getBottomText()}
```

Comment:
nav.footer.bottomTextStart is defined for all languages. I think it should be 
len(nav.footer.bottomTextStart) > 0 here